examples/waspello/src/cards/MainPage.jsx

```diff
@@ -127,13 +127,29 @@ const MainPage = ({ user }) => {
               </div>
             </PositionProvider>
           )}
-        </Droppable>
+        </StrictModeDroppable>
       </DragDropContext>
 
     </UserPageLayout>
   )
 }
 
+export const StrictModeDroppable = ({ children, ...props }) => {
```

Comment:
Naming this `StrictModeDroppable` feels like it leaks the implementation details - no need for the outside user to know that this is specific for the strict mode :) 

I'd maybe suggest, importing the `Droppable` with an alias e.g. `import { Droppable as DroppableComponent } from ...` and then naming this simply `Droppable`. 

Maybe go even further and extra this into a separate file called `Droppable.jsx` from which we can import this specialised component.

Reply:
addressed the comment, moved it to a separate file.